tests/src/Kernel/Element/Select2Test.php

```diff
@@ -59,6 +59,17 @@ public function testEmptyOption() {
     ];
     $this->render($select);
     $this->assertOptionNotExists('field-foo', '');
+
+    // #empty_value and #empty_option properties check
+    $select = [
+      '#type' => 'select2',
+      '#options' => [1 => 'One', 2 => 'Two'],
+      '#empty_option' => 'zero',
+      '#empty_value' => 'Zero option value',
+      '#attributes' => ['data-drupal-selector' => 'field-foo'],
```

Comment:
I think you have to add at least the key #required here as well

Reply:
done